src/main/java/com/epam/ms/service/DefaultNutritionProgramService.java

```diff
@@ -55,4 +74,10 @@ private void copyProgramData(DefaultNutritionProgram from, DefaultNutritionProgr
         to.setCalories(from.getCalories());
         to.setNumberOfDays(from.getNumberOfDays());
     }
+
+    private void sendEventToQueue(String eventKey, DefaultNutritionProgram program) throws JsonProcessingException {
```

Comment:
I'd also propose to have a dedicated class that will be responsible for working with the Queue.

Reply:
Thank you, done